main.py

```diff
@@ -0,0 +1,63 @@
+import json
+import os
+
+from fastapi import FastAPI, Depends, HTTPException
+from fastapi.middleware.cors import CORSMiddleware
+from fastapi.security.api_key import APIKeyHeader, APIKey
+from starlette.status import HTTP_403_FORBIDDEN
+
+from nodeodm_client import NodeodmClient
+
+app = FastAPI()
+
+origins = [
+    "*"
+]
+
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=origins,
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
+
+
+with open('config.json', 'r') as file:
+    config = json.load(file)
+
+API_KEY = config['API_KEY']
+IMAGE_DIR = config['IMAGE_DIR']
+API_KEY_NAME = "access_token"
+api_key_header = APIKeyHeader(name=API_KEY_NAME, auto_error=False)
+
+
+async def get_api_key(
+        api_key_header: str = Depends(api_key_header)
+):
+    if api_key_header == API_KEY:
```

Comment:
Würde bedeuten, dass Groß- und Kleinschreibung egal ist. Ist das so gewollt?

https://www.ionos.com/digitalguide/websites/web-development/how-to-compare-strings-in-python/

Reply:
Also laut dem Link würde er Groß und Kleinschreibung berücksichtigen. Habe es eben getestet:
Als Key habe ich "API_KEY" verwendet.
Key = "api_key" gibt ein forbidden.